util/package.go

```diff
@@ -159,7 +174,43 @@ func NewPackage(basePath string) (*Package, error) {
 		if p.PolicyTemplates[i].Multiple == nil {
 			p.PolicyTemplates[i].Multiple = &trueValue
 		}
+
+		// Collect basic information from policy templates and store into the /search endpoint
+		if p.PolicyTemplates != nil {
+			t := p.PolicyTemplates[i]
+			baseT := BasePolicyTemplate{
+				Name:        t.Name,
+				Title:       t.Title,
+				Description: t.Description,
+			}
+
+			for k, i := range p.PolicyTemplates[i].Icons {
+				t.Icons[k].Path = i.getPath(p)
+			}
+
+			baseT.Icons = t.Icons
+			p.BasePolicyTemplates = append(p.BasePolicyTemplates, baseT)
+		}
+
+		// Store paths for all screenshots under each policy template
+		if p.PolicyTemplates[i].Screenshots != nil {
+			for k, s := range p.PolicyTemplates[i].Screenshots {
+				p.PolicyTemplates[i].Screenshots[k].Path = s.getPath(p)
+			}
+		}
+
+		// Store policy template specific README
+		readmePath := filepath.Join(p.BasePath, "docs", p.PolicyTemplates[i].Name+".md")
+		readme, err := os.Stat(readmePath)
+		if err == nil && readme != nil {
```

Comment:
I'm not fan of swallowing errors. Maybe let's check if it's the error for non existing file, otherwise report an error (as it's not expected).

Reply:
Will do!